package.json

```diff
@@ -8,7 +8,7 @@
   "dependencies": {
     "@centerforopenscience/list-of-licenses": "1.0.0",
     "@centerforopenscience/markdown-it-toc": "~1.1.1",
-    "@centerforopenscience/osf-style": "1.6.0",
+    "@centerforopenscience/osf-style": "https://github.com/CenterForOpenScience/osf-style#develop",
```

Comment:
Ideally, this should pin to a new version number, but failing that, to a specific commit. Pinning to a branch `HEAD` makes it unstable.

Reply:
I was expecting a release to be cut when this was merged, but doesn't look like that happened. Do you know who holds the keys to osf-style? At least one of us should probably have access?

Reply:
Looks to be @jamescdavis

Reply:
Talked to @jamescdavis about this.  After this passes QA, then there'll be a new `osf-style` release with these changes.  But until then, the repos need to be pointed at the specific commit. 

I'll update this and the other repos real quick to point to the commit instead of `develop` branch